### ISSUE TITLE
Wrong fingerprint shown for intermediary firmware when installing out-of-the-box Trezor

### ISSUE THREAD

**Issue by matejcik** [trezor/trezor-suite#4156]
Per [this report](https://www.reddit.com/r/TREZOR/comments/p43xn0/mismatched_firmware_fingerprints_on_brand_new/h8wariv/?utm_source=reddit&utm_medium=web2x&context=3), the fingerprint shown in Suite is for firmware 1.6.3, but what's sent to Trezor is actually the intermediary firmware.
[suite fingerprint](https://jumpshare.com/v/Coxbn5blhEtL1pSrTT85): e8dbb4b7fe8384afd4c99790277c2f2f366a1a0f3957aa3545c75371a99a8fcc
[Trezor fingerprint](https://jumpshare.com/v/IMduurLvImGFvmUecK4Q): ba37d1523ec4cf003b5e28b2fe50a380d694bf53c99d3d4c7a7ffb0d93984d6c

I don't have the hardware to reproduce this personally.

**Comment by hiviah** [comment]
I'd guess this might the part causing this:

> I just opened a new Trezor One that I purchased about 4 years ago as a backup

I have no idea where it was bought or from which warehouse was it shipped from, but it might have containted very old bootloader. Maybe even older than the intermediate FW supports.

**Comment by matejcik** [comment]
doesn't the intermediary firmware support _all_ bootloaders?

**Comment by bosomt** [comment]
@matejcik it can be related to same issue we created about outdated changelog/fw version ?
https://github.com/trezor/trezor-suite/issues/4074

**Comment by matejkriz** [comment]
> @matejcik it can be related to same issue we created about outdated changelog/fw version ? #4074

yes, it's basically the same issue

**Comment by STew790** [comment]
@matejkriz if it is basically the same issue, should we close this as well since #4074 is fixed already?

**Comment by matejkriz** [comment]
> @matejkriz if it is basically the same issue, should we close this as well since #4074 is fixed already?

nope, almost the same changes needs to be done for fignerprint too...

**Comment by STew790** [comment]
**QA NOK**
Unfortunately, I just updated from 1.5.2 FW to 1.10.3 and the issue is that only the device shows the fingerprint and not the Suite. This is due to this issue https://github.com/trezor/trezor-suite/issues/4224.

**Info:**
 - Suite version: web 21.12.1 (e6e9041850c1bdddfcc6590b1f4e6a47c8a40ce8)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: model 1 1.10.3 regular

**Comment by matejkriz** [comment]
@STew790 Could you please try it from FW 1.6.0 or 1.6.1 to verify, that the fingerprint is OK in case it's shown? As you mentioned, not showing the fingerprint in Suite is another issue 😞

**Comment by matejkriz** [comment]
The verification is blocked by https://github.com/trezor/trezor-suite/issues/4224

**Comment by chrisguida** [comment]
Can confirm that this is still happening. Seeing the wrong fingerprint is scary for someone setting up a new Trezor. The fingerprint in [this comment](https://forum.trezor.io/t/finger-prints-dont-match-during-the-firmware-update/1932/2) is indeed what was displayed on the Trezor's screen. Trezor Suite should be updated to display the fingerprint for this "intermediary firmware", and should notify the user that this is what is happening.

**Comment by STew790** [comment]
**QA NOK**

I just had this issue again on latest version when updating FW from 1.4.0 to 1.11.1.

Fingerprint shown on device:
![image](https://user-images.githubusercontent.com/48477147/180779919-62d637a6-ae3b-46b5-8779-a0c3cfad7482.png)

Fingerprint shown in Suite:
![image](https://user-images.githubusercontent.com/48477147/180779848-67583486-01a9-4d00-94a3-2de40ba3b383.png)

**Info:**
 - Suite version: desktop 22.7.3 (8eddb2accd31c78577f276df2e9ecf773d2c3c17)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.7.3 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: 
 - Transport: bridge 2.0.31

**Comment by matejcik** [comment]
another user report: https://forum.trezor.io/t/1st-time-model-1-use-fingerprint-doesnt-match/12248/6

**Comment by chrisguida** [comment]
Can you guys please at least just post the correct intermediary firmware fingerprint? Not having it posted anywhere totally defeats the purpose of displaying the fingerprint.

**Comment by matejcik** [comment]
no problem!
at version 1.12.1, this is:
```
2CDB86EA7ACCA0E4
8A063B48CA22679C
A9093EC7CDA9FA56
1FFC322CBA7C3149
```

**Comment by matejkriz** [comment]
This hasn’t been fixed, but since it isn’t blocking and the issue has been inactive for a long time, we’re closing it to keep the backlog manageable. Feel free to reopen it if you encounter this issue again.
